### PR TITLE
fix: Align right header element correctly

### PR DESCRIPTION
**Pull request by olerichter00**
Resolves https://www.notion.so/artsy/Manage-saves-font-looks-broken-on-the-offer-page-update-font-size-according-to-design-and-center-bu-b344bd5a99f941fea5df5d4048679090?pvs=4

- Related Eigen PR: https://github.com/artsy/eigen/pull/9928

### Description

This pull request corrects the text alignment and centering in the screen header component.

| Before | After |
| --- | --- |
|Simulator Screenshot - iPhone 15 Pro - 2024-03-04 at 16 10 10 | Simulator Screenshot - iPhone 15 Pro - 2024-03-04 at 15 55 22|

<details>
  <summary>📦 Published PR as canary version: <code>13.1.18--canary.200.1559.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @artsy/palette-mobile@13.1.18--canary.200.1559.0
  # or 
  yarn add @artsy/palette-mobile@13.1.18--canary.200.1559.0
  ```
</details>

### CODE REVIEW

File: src/elements/Skeleton/Skeleton.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ export const SkeletonText: FC<TextProps> = ({ children, ...rest }) => {
 
   return (
     <Flex alignSelf="flex-start">
-      <Text {...rest} bg={color("black10")} color={color("black10")}>
+      <Text {...rest} bg={color("black10")} color={color("rgba(0, 0, 0, 0)")}>
```
Comment (damassi) [inline src/elements/Skeleton/Skeleton.tsx:38]: This breaks darkmode

File: src/elements/Screen/Header.tsx [hunk 2]
```diff
@@ -82,14 +105,15 @@ const Center: React.FC<{
...
+  const { width: screenWidth } = useScreenDimensions()
 
   if (hideTitle) {
     return null
   }
 
   if (!animated) {
     return (
-      <Flex flex={1} flexDirection="row" width="100%">
+      <Flex position="absolute" left={0} flexDirection="row" minWidth={screenWidth}>
```
Comment (MounirDhahri) [inline src/elements/Screen/Header.tsx:116]: did position `absolute` with a defined `width` not work here?

**Review by damassi — CHANGES_REQUESTED** [review]
I have a sneaking suspicion that this change might break Folio, which has a different manner of of title behavior than eigen (mostly left aligned, up to the width of the left element). Can we achieve the same thing by remaining with flexbox? As soon as things become manually positioned we lose all manner of flexibility. 

Also, not sure i'm seeing the issue in the screenshots.

**Review by MounirDhahri — APPROVED** [review]
LGTM, I am sure you already did it, but I would only suggest to test this in different screens because it's being used not only in Eigen screens but also in Folio

File: src/elements/Skeleton/Skeleton.tsx (hunk 1 above)
Comment (olerichter00) [reply src/elements/Skeleton/Skeleton.tsx]: Thanks! This change isn't releated to the fix, forgot to remove it before pushing my changes 🙄

**Comment by olerichter00** [comment]
> I have a sneaking suspicion that this change might break Folio, which has a different manner of of title behavior than eigen (mostly left aligned, up to the width of the left element). Can we achieve the same thing by remaining with flexbox? As soon as things become manually positioned we lose all manner of flexibility.
Also, not sure i'm seeing the issue in the screenshots.

Thanks for the reviews 🙏 

Before this fix, we had to set the width of the right element manually to ensure it is rendered correctly ([Eigen code](https://github.com/artsy/eigen/blob/adada5761a116f314f21a0ce60453c616fd285a4/src/app/Scenes/Activity/components/PartnerOfferCreatedNotification.tsx#L46)).

I just realized that what I'm trying to achieve with `position: absolute` would also work by setting `flex: 1` for both the left and right elements to ensure they have the same size, positioning the title exactly in the center of the screen ([JSFiddle with CSS example](https://jsfiddle.net/olerichter00/7ru59kzo/))💡 I'll update this PR and get rid of the `position: absolute`.

**Comment by damassi** [comment]
Sweet, that seems like a good fix!

**Comment by olerichter00** [comment]
Closing this PR in favor of https://github.com/artsy/palette-mobile/pull/203

**Pull request closed without merging**